ui/main/src/app/store/effects/processes.effects.ts

```diff
@@ -0,0 +1,42 @@
+/* Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * See AUTHORS.txt
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ * This file is part of the OperatorFabric project.
+ */
+
+import {Injectable} from '@angular/core';
+import {Actions, createEffect, ofType} from '@ngrx/effects';
+import {merge, Observable, of, timer} from 'rxjs';
+import {ProcessesActionTypes} from '@ofActions/processes.actions';
+import {catchError, debounce, map} from 'rxjs/operators';
+import {ProcessesService} from '@ofServices/processes.service';
+
+@Injectable()
+export class ProcessesEffects {
+    constructor(private actions$: Actions, 
+        private service: ProcessesService) {
+    }
+
+    updateBusinessConfig: Observable<any> = createEffect(() => this.actions$
+        .pipe(
+            ofType(ProcessesActionTypes.BusinessConfigChange),
+            debounce(() => timer(10000)),
+            map(() => {
+                merge(
+                    this.service.loadAllProcesses(),
+                    this.service.loadProcessGroups(),
+                    this.service.areTranslationsLoaded()
+                ).subscribe(() => {});
+                    return of();
+                }
+            ),
```

Comment:
```suggestion
                        map(() => {
                    this.service.loadAllProcesses().subscribe();
                    this.service.loadProcessGroups().subscribe();
                }
            ),
```
Since we don't want to do any action once the loading is complete, I'm not sure the `merge` is necessary here, but maybe I'm missing the point?
Also, it looks like we can do without the subscription to `areTranslationsLoaded` as the loading of translations is triggered as part of `loadAllProcesses`, and we don't care about it being finished or not since we're not doing anything afterwards.
I've tried the above code and I looked like it worked on "passing" cases, but I don't know about error management as I haven't been able to provoke an error.

Reply:
Done